### PR TITLE
Add yyets v2.54

### PR DESCRIPTION
**Pull request by Maecenas**
[yyets/人人影视/字幕组](http://www.zimuzu.io)
> yyets is the subtitles distribution software of YYeTs that offers translation of movies, TV series and open courses since 2004

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
>
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/rrshare.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask 'rrshare' do
+  version '2.53'
+  sha256 '4a7bcc893bf7bd492ba6a00f72809e7f597722db0ba81fcfab88fc762a3b4246'
+
+  # appdown.rrys.tv was verified as official software distribution when first introduced to the cask
+  url "http://appdown.rrys.tv/YYetsShare.#{version}.dmg"
+  name '人人影视'
```
Comment (vitorgalvao) [inline Casks/rrshare.rb]: Please add the english name on top ([like so](https://github.com/Homebrew/homebrew-cask/blob/0fe48607f5656e4f1de58c6884945378b7e6f960/Casks/cave-story.rb#L7#L9)).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/rrshare.rb (hunk 1 above)
Comment (Maecenas) [reply Casks/rrshare.rb]: Sure! Already added the English name. By the way, should I squash all the commits before merging or let GitHub do this for me?

**Comment by vitorgalvao** [comment]
I’m confused. Is it called `yyets` or `rrshare`?

**Comment by Maecenas** [comment]
It's also confusing to me. As when I followed the [Cask Token Reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md#converting-to-ascii) and found that this app was named after `rrshare` by developer. However, there is no other exposure to this name, and not even on its homepage. What's more, searching `rrsahre` with major search engines would only give unrelated messages. The organization `YYeTs` however, is the consistent name for referencing and searching.

I personally recommend using `yyets` or `YYeTs` to reference the task for searching/indexing purposes. It would be better if we add the Chinese annotation/cask name `人人影视` (`YYeTs`'s Chinese name) to it.  

```plist
<!-- Digest of 人人影视.app/Info.plist  -->

	<key>CFBundleExecutable</key>
	<string>rrshare</string>
        <key>CFBundleName</key>
        <string>人人影视</string>
	<key>CFBundleIdentifier</key>
	<string>com.yyets.rrshare</string>
```
@vitorgalvao

**Review by vitorgalvao — APPROVED** [review]

**Comment by Maecenas** [comment]
Squash previous commits. However, if the related branch name were to change, this PR is set to track only the original one. Would it be better to close this PR and reference it from a new one?

**Comment by vitorgalvao** [comment]
There’s no need to squash commits. Github’s web interface let’s us do it on merge.

**Pull request merged**